src/jit/gtlist.h

```diff
@@ -145,6 +145,10 @@ GTNODE(LT               , "<"            ,GenTreeOp          ,0,GTK_BINOP|GTK_RE
 GTNODE(LE               , "<="           ,GenTreeOp          ,0,GTK_BINOP|GTK_RELOP)
 GTNODE(GE               , ">="           ,GenTreeOp          ,0,GTK_BINOP|GTK_RELOP)
 GTNODE(GT               , ">"            ,GenTreeOp          ,0,GTK_BINOP|GTK_RELOP)
+#ifndef LEGACY_BACKEND
```

Comment:
It probably doesn't matter, but these could potentially be declared under
`#if defined(_TARGET_XARCH_) && !defined(LEGACY_BACKEND)`

---
In reply to: [96982872](https://github.com/dotnet/coreclr/pull/8672#discussion_r96982872) [](ancestors = 96982872)